src/JsonSchema.Tests/GithubTests.cs

```diff
@@ -960,6 +960,108 @@ public void Issue600_BaseDocumentOutputSchemaLocation()
 		Assert.That(targetSchemaLocation.OriginalString, Is.EqualTo("http://localhost/v1#/components/schemas/user/properties/last-name"));
 	}
 
+	[Test]
+	public void Issue881_DateTimeValidationPassesWithOneOfSchema()
+	{
+		var file = GetFile(881, "schema");
+
+		var schema = JsonSchema.FromFile(file);
+
+		var instance = new JsonObject()
+		{
+				new KeyValuePair<string, JsonNode?>("oneOfDates", "2025-01-02T23:03:22.222Z")
+		};
+
+		var result = schema.Evaluate(instance, new EvaluationOptions { OutputFormat = OutputFormat.List });
+
+		result.AssertValid();
+	}
+
+	[Test]
+	public void Issue881_DateValidationPassesWithOneOfSchema()
+	{
+		var file = GetFile(881, "schema");
+
+		var schema = JsonSchema.FromFile(file);
+
+		var instance = new JsonObject()
+		{
+				new KeyValuePair<string, JsonNode?>("oneOfDates", "2025-01-02")
+		};
+
+		var result = schema.Evaluate(instance, new EvaluationOptions { OutputFormat = OutputFormat.List });
+
+		result.AssertValid();
+	}
+
+	[Test]
+	public void Issue881_DateValidationFailsAgainstDateTime()
+	{
+		var file = GetFile(881, "schema");
+
+		var schema = JsonSchema.FromFile(file);
+
+		var instance = new JsonObject()
+		{
+				new KeyValuePair<string, JsonNode?>("dateWithDateFormat", "2025-01-02T23:03:22.222Z")
```

Comment:
```suggestion
				["dateWithDateFormat"] = "2025-01-02T23:03:22.222Z"
```

Please update all of these.